presidio-analyzer/setup.cfg

```diff
@@ -1,2 +1,4 @@
 [flake8]
 max-line-length = 88
+
+extend-ignore = D100,D101,D102,D104,D105,D107,D103,D200,D202,D205,D208,D210,E251,D400,D401,D403,F401,W503,N803
```

Comment:
remove once analyzer will be aligned with flake8 (mainly docstrings)

Reply:
Is there a user-story on this?

Reply:
Yes